### PR TITLE
Bw 575

### PR DESCRIPTION
**Pull request by milansar**

### CODE REVIEW

File: projects/trivia/src/app/components/app/app.component.tns.ts [hunk 1]
```diff
@@ -117,7 +126,7 @@ export class AppComponent implements OnInit, OnDestroy {
 
     const alertOptions = {
       title: 'New version available',
-      message: 'Please, update app to new version to continue reposting.',
+      message: 'Please, update app to new version to continue posting.',
```
Comment (milansar) [inline projects/trivia/src/app/components/app/app.component.tns.ts]: change message to "New version available, please update to new version to continue"

File: projects/shared-library/src/lib/core/services/utils.tns.ts [hunk 2]
```diff
@@ -14,22 +15,30 @@ export class Utils extends UtilsCore {
...
-      break;
+        break;
     }
     this.messageConfig.message = message;
     this.message.show(this.messageConfig);
 
   }
 
+  sendErrorToCrashlytics(type: any, error: any) {
+    firebase.crashlytics.sendCrashLog({
```
Comment (anihalaney) [inline projects/shared-library/src/lib/core/services/utils.tns.ts]: is there a way to enable/disable crashlytics using a setting?

File: .gitignore [hunk 3]
```diff
@@ -71,3 +71,5 @@ certificates/*
 release/*
 /**/*.apk
 /**/*.ipa
+
+package-lock.json
```
Comment (anihalaney) [inline .gitignore]: why do we ignore package-lock?

Comment (milansar) [reply .gitignore]: remove package-lock from gitignore

File: projects/trivia/src/app/components/app/app.component.tns.ts [hunk 4]
```diff
@@ -85,14 +89,22 @@ export class AppComponent implements OnInit, OnDestroy {
...
   }
 
   async checkForceUpdate() {
 
     let version;
     try {
-        version = await appversion.getVersionCode();
+      version = await appversion.getVersionCode();
     } catch (error) {
+      this.utils.sendErrorToCrashlytics('appLog', new java.lang.Exception("test"));
```
Comment (milansar) [inline projects/trivia/src/app/components/app/app.component.tns.ts]: remove this test log and replace with actual error

**Pull request merged**